src/Symfony/Component/Workflow/composer.json

```diff
@@ -26,11 +26,14 @@
     "require-dev": {
         "psr/log": "~1.0",
         "symfony/dependency-injection": "~3.4|~4.0",
-        "symfony/event-dispatcher": "~3.4|~4.0",
+        "symfony/event-dispatcher": "^4.3",
         "symfony/expression-language": "~3.4|~4.0",
         "symfony/security-core": "~3.4|~4.0",
         "symfony/validator": "~3.4|~4.0"
     },
+    "conflict": {
+        "symfony/event-dispatcher": "<4.3"
```

Comment:
why is this needed? the minimum version is already `^4.3`

Reply:
because require-dev doesn't apply to the root app